src/test/java/org/mule/modules/watsonvisualrecognition/automation/functional/CreateClassifierTestCases.java

```diff
@@ -19,17 +18,19 @@ public void testWithNullFile() throws VisualRecognitionException {
 	}
 
 	@Test
-	public void testWithFile() {
-		VisualClassifier classifier = null;
+	public void verifyCreation() {
+		VisualClassifier dummyClassifier = null;
+		VisualClassifier expectedClassifier = null;
 		try {
-			classifier = getConnector().createClassifier(new File(TestDataBuilder.sampleZipPath()), "test_class", "Test Classifier", new File(TestDataBuilder.negativeSampleZipPath()));
+			dummyClassifier = getConnector().createClassifier(new File(TestDataBuilder.sampleZipPath()), 
+					"test_class", "Test Classifier", 
+					new File(TestDataBuilder.negativeSampleZipPath()));
+			String classifierId = dummyClassifier.getId();
+			expectedClassifier = getConnector().retrieveClassifierDetails(classifierId);
```

Comment:
Just validate the response for the creation operation, don't spent another request retrieving the classifier, we assume that the Watson service works well.